_i18n/en.yml

```diff
@@ -19,6 +19,7 @@ footer:
   contact: Contact
   partners: Partners
 stories:
+  intro: Our biggest commitment as a project is to make public data accessible to the anyone. Read below our independent productions based on what we work.
```

Comment:
As a project, our biggest commitment is to make public data accessible to the anyone. Following are our independent production based on the work we do.